include/dsCompositeIn.h

```diff
@@ -252,6 +253,21 @@ dsError_t dsCompositeInScaleVideo (int32_t x, int32_t y, int32_t width, int32_t
  */
 typedef void (*dsCompositeInConnectCB_t)(dsCompositeInPort_t Port, bool isPortConnected);
 
+/**
+ * @brief Notifies applications when the Composite input port video mode changes
+ *
+ * HAL Implementation must call this method to deliver updated Composite input port video mode info
+ * to the Caller
+ *
+ * @param[in] port              - Port in which video mode updated. Please refer ::dsCompositeInPort_t
```

Comment:
just thinking, as these parameters used as output from HAL, should we make them as [out] param rather than in.  Let's discuss this.

Reply:
updated

Reply:
Can we consider to update this on all the callbacks please?

Reply:
This is not addressed